src/wire/sixlowpan.rs

```diff
@@ -910,7 +910,7 @@ pub mod iphc {
         ) -> usize {
             self.set_cid_field(0);
             self.set_sac_field(0);
-            self.set_sam_field(0b11);
+            self.set_sam_field(0b00);
```

Comment:
This should be removed, right? Every path below sets it.
@thvdveld

Reply:
Yes indeed, however, I think we can leave it. It's better to first set everything to zero before modifying the fields. And the compiler will optimize that away, probably.

Reply:
I wouldn't rely on that. And even then you can remove it in both L917 and L961.
Anyway. Just a minor nit.

Reply:
Removed it in 75bacb1e46b5e5a0fa0c758ebbc7bbc963fb5bf3 just to be sure :wink: